packages/server/utils/analytics/helpers.ts

```diff
@@ -23,6 +23,7 @@ export const createMeetingProperties = (
     meetingTemplateName: template?.name,
     meetingTemplateScope: template?.scope,
     meetingTemplateIsFromParabol: template?.isStarter,
+    meetingTemplateIsFree: template?.isFree,
```

Comment:
-1 Would prefer a default `false` over `undefined` to make analysis in Amplitude easier to understand, e.g., [Aviva recently was confused](https://parabol.slack.com/archives/C03C3J42WDU/p1666296869022669). While we are here, can we also fix the line above:

```suggestion
    meetingTemplateIsFromParabol: template?.isStarter ?? false,
    meetingTemplateIsFree: template?.isFree ?? false,
```